assets/src/stories-editor/index.js

```diff
@@ -312,6 +313,7 @@ addFilter( 'blocks.registerBlockType', 'ampStoryEditorBlocks/filterBlockTransfor
 addFilter( 'editor.BlockEdit', 'ampStoryEditorBlocks/addStorySettings', withAmpStorySettings );
 addFilter( 'editor.BlockEdit', 'ampStoryEditorBlocks/addPageNumber', withPageNumber );
 addFilter( 'editor.BlockEdit', 'ampStoryEditorBlocks/addEditFeaturedImage', withEditFeaturedImage );
+addFilter( 'editor.BlockEdit', 'ampEditorBlocks/addVideoBlockPreview', withVideoBlockPreview, 9 );
```

Comment:
The priority of `9` allows the other filters to run, which add 'Block Position,' 'Animation,' and 'Story Settings' to the Inspector.